### DISCUSSION TITLE
Brownbags information pipeline addition

### DISCUSSION THREAD

**Discussion by bogdaaamn** [Virtual-Coffee/VC-Community-Docs#163] [Lunch & Learns]
Hello @Virtual-Coffee/maintainers and @meg-gutshall and @Virtual-Coffee/audio-visual-team and all the Brownbags folks!

I know there is a whole recurring discussion about the structure of a Brownbag template in PR #125, but I have some content features I would like to discuss. I see a lot of changes in the [`brownbag--event.md`](https://github.com/Virtual-Coffee/VC-Contributors/pull/125/files#diff-70d34fd26b21cbb8d6fa12bed5e2259e53bbe1ebc874ba82abc0cfbcba45b380) template file, but I know that the [Brownbag form](https://virtualcoffee.io/brownbag-idea/) and the bot that creates the issues out of that form (cc @danieltott) have different outcomes. 

- **Speaker bio and social media**: I believe that the bios are very important for building SEO for each speaker when we post the Brownbag videos. They're currently missing from the form and from the Brownbag issues created (_eg_ brownbag [#115](https://github.com/Virtual-Coffee/VC-Contributors/issues/115)). Currently, the AV Team is getting them from the descriptions of past events like Lightning Talks, but they're not always available. 
    
    I know we discussed that we want to keep the Brownbag form as simple as possible, so I understand if we don't include them, but what do you think it would be a nice way to source them? Or should we drop them altogether? Or include them in the pipeline and let people write it down in the Brownbag form when the submit a new idea?
    
    I am also thinking of getting them from the members' description on the VC website, but they are not always suitable for the format that we aim to write in the description of the Brownbags. For example, @BekahHW's bio that we currently use vs GitHub/VC website bio: 
    
    > Bekah graduated from the Flatiron School Software Engineering program in May of 2019 and since then she has started a consultancy specializing in front-end development, created a developer community called Virtual Coffee, stayed active on Twitter and in tech-related slack groups, spoke on podcasts and at conferences, and has continued to mom her four kids. Prior to tech, she spent 10 years teaching college English, three years organizing events and community, and fully embraces being a writer of many genres: code, non-fiction, screenplay, poetry, curriculum.
    
    > Community + Code | Creator, Maintainer Virtual-Coffee | Frontend Dev bekahhw.github.io/ 
    
- **The date of a Brownbag**: Currently the data  (_eg_ brownbag [#115](https://github.com/Virtual-Coffee/VC-Contributors/issues/115)) includes the date that the speaker is available (as an answer to the **What date and time works for you?** from the [Brownbag form](https://virtualcoffee.io/brownbag-idea/)). From the brownbags docs I understand that the final date is confirmed and written down in a Virtual Coffee Google calendar and meetingplace.io, but it never gets to the actual GitHub issue. The AV Team checks meetingplace.io for the actual date of a brownbag, but having it in the issues would benefit both the AV Editors and to sort the Brownbag issues in the boards (_eg_ [Post-production board](https://github.com/Virtual-Coffee/VC-Contributors/projects/6)). 

    Do you think this automation is actually necessary. If so, is there an easy way to include this in the procedure or to automate it? 
    
What do you all think about this?? 

I am very sorry for the long thoughts dump wall of text, I hope it makes sense. I am open up for clarification or summarising the whole think 😂

**Comment by BekahHW** [comment]
@BogDAAAMN - I tried to remember to drop the date and the bio in the issues, but that didn't happen all the time.

I think it's important to keep bio and specific date off of the initial form, only bc this is a "barrier to submission."

I didn't realize the AV team was hunting down the info, so I'm sorry about that. I should've thought about that. I know Dan was working through a process for that second info gathering that needs to happen, so this is definitely something we're actively looking at.

I do have bios for everyone who's been on the podcast, bc I ask for that in email. I'll update NickyT's, and try to go back through and fill in whoever's I have.

@saramccombs might have some too.

**Reply by kierblk** [reply]
@BogDAAAMN I have been attempting to add the Speaker Bio and photo to the issue to "keep everything together" in one place for you. (I may have missed one or two due to oversight though.) This is a very manual thing I do after getting the event scheduled. I can add their social media links to that process. 

I'm also happy to help gather any speaker information for events that are missing it.

**Comment by bogdaaamn** [comment]
OMG I SEE IT NOW! So sorry @BekahHW and @saramccombs. We were looking at different issues, I see the previous ones are doubled, for exemple #66 and #97. I assume the later ones were created for the post-production purposes! 

Ever since we've changed the procedure, so I will hunt down the doubled ones and close them as we publish them. We will keep using the same issues for both planning and later post-production. The issue will be closed after it gets published on youtube by the AV Team.

Thank you for your work @danieltott @BekahHW @saramccombs @meg-gutshall and I am so sorry I missed it 🤦🏼‍♂️🤦🏼‍♂️🤦🏼‍♂️

**Reply by meg-gutshall** [reply]
Hey @BogDAAAMN, I'm just catching up with all of this today. And yes, that's exactly it. At first Dan was talking about using a parent-child approach where we had a parent issue for each brownbag and child issues (at least that's how I understood it) which is what led to the doubling up that you linked to in #66 and #97.

We've since switched over to a workflow of passing a single issue through the entire lifecycle, cutting out the duplicate work.

I'm looking at the project columns in "Brownbags" and "Post-Production" now and I want to double check that the issues are moving through each one correctly. @danieltott can you verify the table below?

| Brownbags | Post-Production |
| --- | --- |
| Inbox | |
| Scheduled | Inbox |
| Event Completed | Ready for Upload |
| Done | Published |
| Done | Completed |

**I added "Cancelled" columns to each project board as well.*

A new issue labeled "Brownbag: Event" gets created and goes into the "Brownbags" project board's "Inbox" column. Once the workflow is initiated and a date is scheduled, the Moderator moves the issue to the "Scheduled" column of the "Brownbags" project board. At the same time, the issue is automatically labeled "Brownbag: Post-production" and added to the "Inbox" column of the "Post-Production" project board. It continues to move through both boards from there per the table above.

As a reminder, this is what "Completed" means for the Brownbag videos (per the final column in the "Post-Production" project board):
- Edited raw footage
- Completed the "Video details" fields
- Edited the automatic captions
- Added YouTube chapters to video description (if applicable)
- Added link to speaker resources to video description (if applicable)
- Updated original event issue with YouTube video link

Last we left it, we were treating "Published" as our MVP uploads until we could get more people on board to help with the captioning and other details. I defer to Bogdan though on any updates/changes to this process.

Also [these should be the rest of the duplicate issues](https://github.com/Virtual-Coffee/VC-Contributors/issues?q=is%3Aopen+is%3Aissue+author%3Ameg-gutshall+label%3A%22Brownbag%3A+TODO%22+) that I opened before we adopted the new workflow.

**Comment by danieltott** [comment]
@meg-gutshall this all looks right to me - we definitely started with separate issues but @BogDAAAMN and the AV team decided to go with the same issue. I think we can close the duplicates but I'll leave that up to @BogDAAAMN to make sure nothing gets missed.
